docs/examples/MultiSelectSort.tsx

```diff
@@ -1,92 +1,119 @@
+    useSortable({
+      id: props.data.id,
+    });
+  const style = {
+    transform: CSS.Transform.toString(transform),
+    transition,
+  };
 
-function arrayMove<T>(array: readonly T[], from: number, to: number) {
-  const slicedArray = array.slice();
-  slicedArray.splice(
-    to < 0 ? array.length + to : to,
-    0,
-    slicedArray.splice(from, 1)[0]
+  return (
+    <div style={style} ref={setNodeRef} {...attributes} {...listeners}>
+      <components.MultiValue {...props} innerProps={innerProps} />
+    </div>
   );
-  return slicedArray;
-}
+};
 
-const SortableMultiValue = SortableElement(
-  (props: MultiValueProps<ColourOption>) => {
-    // this prevents the menu from being opened/closed when the user clicks
-    // on a value to begin dragging it. ideally, detecting a click (instead of
-    // a drag) would still focus the control and toggle the menu, but that
-    // requires some magic with refs that are out of scope for this example
-    const onMouseDown: MouseEventHandler<HTMLDivElement> = (e) => {
-      e.preventDefault();
-      e.stopPropagation();
-    };
-    const innerProps = { ...props.innerProps, onMouseDown };
-    return <components.MultiValue {...props} innerProps={innerProps} />;
-  }
-);
+const MultiValueContainer = (props: MultiValueGenericProps<ColourOption>) => {
+  const { isOver, setNodeRef } = useDroppable({
+    id: 'droppable',
+  });
 
-const SortableMultiValueLabel = SortableHandle(
-  (props: MultiValueGenericProps) => <components.MultiValueLabel {...props} />
-);
+  const style = {
+    color: isOver ? 'green' : undefined,
+  };
+
+  return (
+    <Tooltip content={'Customise your multi-value container!'}>
+      <div style={style} ref={setNodeRef}>
+        <components.MultiValueContainer {...props} />
+      </div>
+    </Tooltip>
+  );
+};
 
-const SortableSelect = SortableContainer(Select) as React.ComponentClass<
-  Props<ColourOption, true> & SortableContainerProps
->;
+const MultiValueRemove = (props: MultiValueRemoveProps<ColourOption>) => {
+  return (
+    <components.MultiValueRemove
+      {...props}
+      innerProps={{
+        onPointerDown: (e) => e.stopPropagation(),
+        ...props.innerProps,
+      }}
+    />
+  );
+};
 
-export default function MultiSelectSort() {
-  const [selected, setSelected] = React.useState<readonly ColourOption[]>([
+const MultiSelectSort = () => {
+  const [selected, setSelected] = React.useState<ColourOption[]>([
     colourOptions[4],
     colourOptions[5],
   ]);
 
   const onChange = (selectedOptions: OnChangeValue<ColourOption, true>) =>
-    setSelected(selectedOptions);
+    setSelected([...selectedOptions]);
 
-  const onSortEnd: SortEndHandler = ({ oldIndex, newIndex }) => {
-    const newValue = arrayMove(selected, oldIndex, newIndex);
-    setSelected(newValue);
-    console.log(
-      'Values sorted:',
-      newValue.map((i) => i.value)
-    );
+  const onSortEnd = (event: DragEndEvent) => {
```

Comment:
`onSortEnd` should be a `useCallback` function, like this (and maybe rename it `onDragEnd`: 

```js
const onDragEnd = useCallback((event: DragEndEvent)=> {}, [])
```

Reply:
Is there any way I can do this? I tried to get it done in @codercodingthecode repository but i don't have permissions, neither in this repository.

Reply:
Open a pr against that repo and we’ll update here.

Reply:
done! #https://github.com/codercodingthecode/react-select/pull/1